client/logit_client.go

```diff
@@ -0,0 +1,270 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/chzyer/readline"
+	gpt_bpe "github.com/wbrown/novelai-research-tool/gpt-bpe"
+	novelai_api "github.com/wbrown/novelai-research-tool/novelai-api"
+	"log"
+	"strings"
+	"unicode"
+    "os"
+    "os/exec"
+)
+
+//var f embed.FS
+const colorGrey = "\033[38;5;240m"
+const colorDkGrey = "\033[38;5;235m"
+const colorWhite = "\033[0m"
+const colorInput = "\033[30;47m"
+
+
+type Adventure struct {
```

Comment:
Is this still an `Adventure`? Seems that we could probably pull this out into a `Client` struct used by both.